src/gc/gc.cpp

```diff
@@ -37212,6 +37212,23 @@ void GCHeap::DiagWalkObject (Object* obj, walk_fn fn, void* context)
     }
 }
 
+void GCHeap::DiagWalkObject2 (Object* obj, walk_fn2 fn, void* context)
+{
+    uint8_t* o = (uint8_t*)obj;
+    if (o)
+    {
+        go_through_object_cl (method_table (o), o, size(o), oo,
+                                    {
```

Comment:
Nit: whitespace is funky

Reply:
This is how the whitespace is for DiagWalkObject, so if you want I can change it, but it looks nice when you're reading it in gc.cpp